src/core/entities/MRTextEntity.js

```diff
@@ -40,10 +40,10 @@ export class MRTextEntity extends MRDivEntity {
 
     /**
      * @function
-     * @description (async) sets up the textObject of the text item.
+     * @description Helper for anytime text is manually touched - in connected and by use of `innerText`.
+     * Handles setting `innerText` like expected, sanitizing it for our use, and also making sure the updates are called.
      */
-    async connected() {
-        await super.connected();
+    _textWasManuallyUpdated() {
```

Comment:
Why is this needed? Aren't we checking `entity.textContent == entity.textObj.text` every frame?

Reply:
nope, we had removed that back when text updating was a big problem - right now we're only checking it when an overall [`eventUpdate` is triggered](https://github.com/Volumetrics-io/mrjs/blob/main/src/core/componentSystems/TextSystem.js#L105) - the `TextSystem`'s update call is empty since it was too costly per frame: https://github.com/Volumetrics-io/mrjs/blob/main/src/core/componentSystems/TextSystem.js#L137

also in the [incoming pr](https://github.com/Volumetrics-io/mrjs/pull/546) later where i rework some of the text system, i bring more of it into the main changes, but i still keep that update empty - didnt want to add all the system changes to this pr as even though it introduces readability and some fixes related to this - a lot of them arent needed until it gets merged

Reply:
hmmm I still think there might be a need for some stuff in the main update. particularly dynamic CSS updates, which we can't detect changes to via events. 

This is good for now but dynamic CSS changes is a table stakes requirement for V1.

Reply:
s.g, dynamic CSS changes we can add to the main update in https://github.com/Volumetrics-io/mrjs/pull/546 since that's just the add of one function call - i'll keep this pr as is

Reply:
new issue #562